src/SqlSquared/Parser/Tokenizer.purs

```diff
@@ -226,7 +226,7 @@ notQuotedIdentOrKeyword = do
     str = S.fromCharArray $ A.cons first other
     low = S.toLower str
   pure if Set.member low keywords
-    then Kw str
+    then Kw str -- we should use low
```

Comment:
Changing `str` to `low` will also fix the issue.

Reply:
Alternatively we could add a sum for the keywords rather than having it contain strings, that way there's no possible way there can be case mismatches etc.

Reply:
Like, keep `Kw`, but have a `data KwToken = ...` inside it rather than `String`

Reply:
I got why this is `str` not `low` --> it's for parsing `:From`, proper fix would be introducing `vari`s to tokenizer not handling `Kw low/str` and `toLower`.

Reply:
> Like, keep Kw, but have a data KwToken = ... inside it rather than String

Unfortunately this will remove ability to parse varis and identifiers 
```
-- ident 
`fRoM`
--varis
:fRoM
```

Reply:
Yeah, this would actually break stuff if we used `low` here @safareli

Reply:
@cryogenian ah ok, yeah, good point 😕